i18n/ru/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,47 @@ sidebar_position: 4
-- *Код переиспользуемый во всех частях приложения - `core`/ `shared` / `app` / ...*
-- *Код бизнес логики - `store` / `model` / `state` / ...*
+## Нейминг в Feature-Sliced Design {#naming-in-fsd}
 
-## Нейминг в FSD {#naming-in-fsd}
+В методологии используются такие специфические термины, как:
 
-Методология оперирует такими терминами, как
+- "app", "process", "page", "feature", "entity", "shared" как имена слоев,
+- "ui', "model", "lib", "api", "config" как имена сегментов.
 
-- `app`, `process`, `page`, `feature`, `entity`, `shared` - *[layers][refs-layers]*
-- `ui`, `model`, `lib`, `api` - *[segments][refs-segments]*
+Оригинальный нейминг очень важен, чтобы предотвратить путаницу среди членов команды и новых разработчиков, присоединяющихся к проекту. Использование уникальных имен также помогает при обращении за помощью к сообществу.
 
-В рамках методологии, каждый из этих терминов имеет [четкое определение][refs-reference]
+## Конфликты нейминга {#when-can-naming-interfere}
 
-Разрабатывая проект по методологии **Feature-Sliced Design** очень важно [придерживаться оригинального нейминга, во избежание недопонимания][disc-src] как среди членов команды, так и за её пределами.
+Конфликты нейминга могут возникать, когда термины, используемые в методологии FSD, пересекаются с терминами, используемыми в бизнесе:
 
-- Если на проект приходит новый разработчик знакомый с методологией, **он должен видеть уже знакомые ему термины**
-- Обращаясь за помощью в комьюнити, вы быстрее получите ответ на ваш вопрос, **если будете использовать ту же терминологию**
+- `FSD#process` vs моделируемый процесс в приложении,
+- `FSD#page` vs страница журнала,
+- `FSD#model` vs модель автомобиля.
 
-## Когда нейминг может помешать? {#when-can-naming-interfere}
+Например, разработчик, увидев слово процесс в коде, будет тратить лишнее время на понимание, о каком процессе идет речь.
 
-Разрабатывая проект, для отображения/построения/моделирования каких либо процессов, или разрабатывая приложение для верстки страниц журнала, вы можете столкнуться с проблемой, что **термины используемые в методологии пересекаются с терминами которыми оперирует ваш бизнес**.
+Эти **коллизии могут нарушить процесс разработки**, поскольку разработчики могут потратить дополнительное время, пытаясь понять, о чем именно идет речь.
 
-- `FSD#process` vs моделируемый процесс в вашем приложении
-- `FSD#page` vs страница журнала
-- `FSD#model` vs модель автомобиля
+Когда глоссарий проекта содержит терминологию, характерную для FSD, крайне важно проявлять осторожность при обсуждении этих терминов с командой и нетехническими заинтересованными сторонами.
 
-<!-- TODO: подумать над примерами для других терминов -->
+Чтобы эффективно общаться с командой, рекомендуется использовать аббревиатуру "FSD" для префиксации терминов методологии. Например, когда речь идет о процессе, можно сказать: "Мы можем поместить этот процесс на уровень features FSD".
```

Comment:
issue: speaking of sticking to terminology :D

also swapped the words so that "FSD" really is a prefix

```suggestion
Чтобы эффективно общаться с командой, рекомендуется использовать аббревиатуру "FSD" для префиксации терминов методологии. Например, когда речь идет о процессе, можно сказать: "Мы можем поместить этот процесс на слой FSD features".
```

Reply:
Fixed in latest commit.